src/build-remote/build-remote.cc

```diff
@@ -137,14 +144,19 @@ static int main_build_remote(int argc, char * * argv)
                 for (auto & m : machines) {
                     debug("considering building on remote machine '%s'", m.storeUri);
 
-                    if (m.enabled
-                        && (neededSystem == "builtin"
-                            || std::find(m.systemTypes.begin(),
-                                m.systemTypes.end(),
-                                neededSystem) != m.systemTypes.end()) &&
-                        m.allSupported(requiredFeatures) &&
-                        m.mandatoryMet(requiredFeatures))
-                    {
+                    if (!m.enabled) {
+                        debug("declined remote machine '%s' because it is disabled", storeUri);
```

Comment:
```suggestion
                        debug("avoiding remote machine '%s' because it is disabled", storeUri);
```

How about avoid for all of these?

The problem with decline is that it seems like the machine is initiating the remote build or something.